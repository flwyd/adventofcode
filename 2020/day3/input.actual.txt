../input/3/input.actual.txt