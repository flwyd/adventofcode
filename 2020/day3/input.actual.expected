part1: 184
part2: 2431272960
